test/bridge.test.ts

```diff
@@ -466,16 +459,14 @@ describe("Bridge", async function () {
     const l2RewardsIndexUsdc = uintFromParts(flushL2Messages[1].payload[6], flushL2Messages[1].payload[7]);
 
     // actually withdraw tokens
-    txDai = await l1Bridge.connect(l1user).withdraw(aDai.address, l2user.starknetContract.address, l1user.address, 28, l2RewardsIndexDai, true);
+    txDai = await l1Bridge.connect(l1user).withdraw(aDai.address, l2user.starknetContract.address, l1user.address, 27n * DAI_UNIT, l2RewardsIndexDai, true);
```

Comment:
any specific reason why we are withdrawing `27 `DAI & `37` USDC ?

Reply:
not really indeed, it used to be the balance of l2 user. now it's just a value below its balance